test/integration/name_conflict.bats

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bats
```

Comment:
What do you think of putting those in api/create.bats and api/rename.bats?

Right now they don't test much